Core/Core/Inbox/AttachmentPicker/AttachmentPickerAssembly.swift

```diff
@@ -17,15 +17,24 @@
 //
 
 import Foundation
+import Combine
 
 public enum AttachmentPickerAssembly {
     public static func makeAttachmentPickerViewController(
+        subTitle: String? = nil,
         env: AppEnvironment = .shared,
         batchId: String,
-        uploadManager: UploadManager
+        uploadManager: UploadManager,
+        alreadyUploadedFiles: CurrentValueSubject<[File], Never>
     ) -> UIViewController {
-        let interactor = AttachmentPickerInteractorLive(batchId: batchId, uploadManager: uploadManager)
-        let viewModel = AttachmentPickerViewModel(router: env.router, interactor: interactor)
+        let interactor = AttachmentPickerInteractorLive(
+            batchId: batchId,
+            folderPath: "conversation attachments",
```

Comment:
I'm not sure what do you mean by that

Reply:
I saw `"conversation attachments"` is used in 3 places as is, and it may be better to put it in a constants somewhere and use that instead.
On doublechecking it, shouldn't the interactor use the passed in `uploadFolderPath` instead of these strings? And/or use this string as default parameter value?